Lib/test/test_largefile.py

```diff
@@ -45,6 +45,20 @@ def tearDownClass(cls):
             raise cls.failureException('File was not truncated by opening '
                                        'with mode "wb"')
 
+    @bigmemtest(size=_2G + 1024 * 1024, memuse=2)
```

Comment:
Are you sure about the memuse=2 parameter? I would only expect memuse=1. Try maybe tracemalloc to get the memory peak: run tests using python3 -X tracemalloc ... and then display tracemalloc.get_traced_memory()[1].

https://docs.python.org/dev/library/tracemalloc.html#tracemalloc.get_traced_memory

(You can run such test on Linux.)